app/Modules/Newsletter/Model/NewsletterService.php

```diff
@@ -66,7 +66,7 @@ class NewsletterService
 	public $linkGenerator;
 
 	/** Path to css file used for css inline of newsletter texts html */
-	const CSS_FILE_PATH = __DIR__ . DIRECTORY_SEPARATOR . '..' . DIRECTORY_SEPARATOR . 'presenters' . DIRECTORY_SEPARATOR . 'templates' . DIRECTORY_SEPARATOR . 'Newsletter' . DIRECTORY_SEPARATOR . 'build.css';
```

Comment:
It's ok on master branch and my local master branch. Interesting :) OK
https://github.com/eventigo/eventigo-web/blob/master/app/modules/newsletter/model/NewsletterService.php#L69